rhosp13/cpe/rhosp13-cpe-dictionary.xml

```diff
@@ -2,7 +2,7 @@
 <cpe-list xmlns="http://cpe.mitre.org/dictionary/2.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:schemaLocation="http://cpe.mitre.org/dictionary/2.0 http://cpe.mitre.org/files/cpe-dictionary_2.1.xsd">
-      <cpe-item name="cpe:/a:redhat:openstack:13">
+      <cpe-item name="cpe:/a:redhat:openstack:13.0">
```

Comment:
@yuumasato this should be `13` and not `13.0`

Reply:
https://nvd.nist.gov/products/cpe search founds only `13.0`

Reply:
Understood, but just like what we do with RHEL, we don't want to be having to constantly update for minor versions.

Reply:
Fixed.

Reply:
Sent a note to the NIST CPE people and copied @yuumasato